src/main/java/org/coffeetrain/doggifs/DogGifRepository.kt

```diff
@@ -0,0 +1,31 @@
+package org.coffeetrain.doggifs
+
+import android.content.Context
+import java.io.InputStream
+import java.util.Arrays
+
+class DogGifRepository(val context: Context) {
+  private val assetGifs = Arrays.asList("pounce", "weiners")
+  private var currentIndex = 0
+
+  fun getGif(handle: String): ByteArray {
```

Comment:
nice, didn't know about operator overloading in kotlin.  I think I'm gonna end up making this asyncronous, will just rename to `loadGif` for now